frontend/simple/views/components/Graphs/PieChart.vue

```diff
@@ -23,6 +25,11 @@
     <div class="c-title">
       <slot></slot>
     </div>
+    <tooltip
+      :text="slices[ephemeral.labelActiveIndex].label"
+      :style="ephemeral.labelStyle"
+      :shouldShow="ephemeral.isLabelVisible"
+    />
```

Comment:
hey @taoeffect I've just noticed I've forgot to implement the feature where on piechart hover to show a tooltip with the label selected. The rule is simple. Show the tooltip at the cursor position on a slice's mouseenter.

![image](https://user-images.githubusercontent.com/14869087/47749589-59ca6e80-dc85-11e8-91ca-356b1bf3f4a3.png)

Reply:
oh cool! 😄